app/javascript/src/controller_default.js

```diff
@@ -138,8 +138,9 @@ function isolatedMethodDeleteLinks() {
 function addExpanderComponents() {
   $("[data-component=Expander]").each(function() {
     var $node = $(this);
+    var $title = $node.find('> h2').first();
     new Expander($node, {
-      title_tag: 'h2',
+      title: $title,
       wrap_content: true,
```

Comment:
`title: $node.find('> h2').first()`

Maybe could also change the original...
`new Expander($node, {`
...to...
`new Expander($(this), {`
...if we doing that, unless one or both variables are used elsewhere.
Might be a little harder (not much) to read the $node change though.